missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/GameWorld.java

```diff
@@ -134,8 +135,9 @@ public void load() {
             Logger.DEBUG.log("Loading new gameworld");
             World world = Bukkit.createWorld(new WorldCreator(worldName));
             Bukkit.getWorlds().add(world);
-            world.setGameRuleValue("doTileDrops", String.valueOf(game.getArena().isDoTileDrops()));
-            world.setGameRuleValue("keepInventory", String.valueOf(game.getArena().isKeepInventory()));
+            
```

Comment:
Does this also work on older versions like 1.13?

Reply:
Should work from 1.13.x - 1.19.x.
https://helpch.at/docs/1.13.1/org/bukkit/World.html#setGameRule-org.bukkit.GameRule-T-